src/site/content/en/blog/bbc-hsts/index.md

```diff
@@ -0,0 +1,78 @@
+
+Recently [Neil Craig](https://twitter.com/tdp_org)&mdash;a lead technical Architect at the BBC&mdash;[tweeted that HTTP Strict Transport Security (HSTS) is being slowly rolled out](https://twitter.com/tdp_org/status/1549028675328573440?s=20&t=S5mm1eEqGUbxZ4jfi-SvtA) for [bbc.com](https://www.bbc.com/). Let's find out what that means for the BBC, and what it could mean for you.
+
+## The problem
+
+Web servers often listen for requests on both ports 80 and 443. Port 80 is for insecure HTTP requests, whereas 443 is for secure HTTPS. This can create a problem, because when you enter an address into your address bar without the `https://` protocol prefix&mdash;like most users tend to do&mdash;some browsers will direct traffic to the insecure HTTP version of a site, for legacy reasons ([though this isn't always the case](https://blog.chromium.org/2021/03/a-safer-default-for-navigation-https.html)).
+
+A common way to ensure users don't access an unsecured version of a website is to place an HTTP-to-HTTPS redirect for all requests. This certainly works, but it kicks off the following chain of events:
+
+1. The server receives a request via HTTP.
+2. The server issues a redirect to go to the HTTPS equivalent of the requested resource.
+3. The server via HTTPS must negotiate a secure connection with the browser.
+4. The content loads as usual.
+
+While redirects work fine, they can be misconfigured in ways that still allow access to the insecure version of a site. Even if everything _is_ configured properly, there is still a security issue in that the user will still connect over insecure HTTP during the redirect phase, which exposes users to the possibility of dangerous [man-in-the-middle attacks](https://en.wikipedia.org/wiki/Man-in-the-middle_attack).
+
+## Enter HSTS
+
+{% BrowserCompat 'http.headers.Strict-Transport-Security' %}
+
+[HSTS](https://developer.mozilla.org/docs/Web/HTTP/Headers/Strict-Transport-Security) is dictated by the `Strict-Transport-Security` HTTP response header for HTTPS requests. When set, return visits to a website will trigger a special redirect known as a "307 Internal Redirect", which is when the browser handles the redirect logic, rather than second it to the server. This prevents the request being intercepted, since it never leaves the browser, so is more secure. As and added bonus, as there is no network request needed to get the redirect, these types of redirect are extremely fast, and so prevent noticeable latency during an HTTP-to-HTTPS hop.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/J9vQLJCLmnHQPb6spsRe.png", alt="A 307 internal redirect from HTTP to HTTPS, triggered by an HSTS header. The 307 redirect only takes 2 milliseconds.", width="800", height="97" %}
+</figure>
+
+Similar in syntax to `Cache-Control`'s `max-age` directive, an HSTS header specifies a `max-age` directive. This directive takes a value in seconds that specifies how long the policy should take effect:
```

Comment:
Your wording would be read as "when the policy will start to take effect", rather than "how long it is effective for".

```suggestion
Similar in syntax to `Cache-Control`'s `max-age` directive, an HSTS header specifies a `max-age` directive. This directive takes a value in seconds that specifies how long the policy is effective for:
```